targets/simple_switch/externs/sym_crypto/README.md

```diff
@@ -0,0 +1,22 @@
+# ADD CRYPTO to SIMPLE_SWITCH
+
+For reference, the extern is implemented in crypto.cpp in this directory.
+This cpp code is to be incorporated in
+behavioral-model/targets/simple_switch/simple_switch.cpp
```

Comment:
We should provide a mechanism for using this code that does not require copying the code to simple_switch.cpp.
My preferred option would be to have a standalone Makefile that can build a .so. The .so can then be imported into simple_switch using `--load-modules`. An alternative, which is not as good IMO (but I have no strong objection) is to just make this part of the bmv2 build system.
I gave some extra information there: https://github.com/p4lang/behavioral-model/issues/697

Reply:
Thinking of this more, adding a separate Makefile and not making it part of the simple_switch build is definitely the right way to go. There is no reason to unconditionally compile and link this example extern to bmv2, and I think the separate Makefile is more representative of the "correct" workflow for someone who wants to add an extern type without changing any of the bmv2 code.

Reply:
I added a Makefile to `behavioral-model/targets/simple_switch/sample_extern/macsec` with a recent commit.